src/sentry/workflow_engine/migrations/0036_action_remove_legacy_fields.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 5.1.5 on 2025-03-07 00:47
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.new_migrations.monkey.fields import SafeRemoveField
+from sentry.new_migrations.monkey.state import DeletionAction
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("workflow_engine", "0035_action_model_drop_legacy_fields"),
+    ]
+
+    operations = [
+        SafeRemoveField(
+            model_name="action",
+            name="target_display",
+            deletion_action=DeletionAction.DELETE,
+        ),
+        SafeRemoveField(
+            model_name="action",
+            name="target_identifier",
+            deletion_action=DeletionAction.DELETE,
+        ),
+        SafeRemoveField(
+            model_name="action",
+            name="target_type",
+            deletion_action=DeletionAction.DELETE,
+        ),
+        migrations.RunSQL(
+            """
+            DROP TABLE "workflow_engine_workflowaction";
+            """,
+            # We just create a fake table here so that the DROP will work if we roll back the migration.
+            reverse_sql="CREATE TABLE workflow_engine_workflowaction (fake_col int)",
+        ),
```

Comment:
https://github.com/getsentry/sentry/pull/78277 - cause i'm a bit lazy, happy to drop this update if it's too much in a single migration too.